emmet-core/emmet/core/structure_group.py

```diff
@@ -193,11 +214,29 @@ def from_ungrouped_structure_entries(
             )
         return results
 
+    @staticmethod
+    def get_host_and_insertion_ids(
+            entries: List[Union[ComputedEntry, ComputedStructureEntry]],
+            ignored_specie: str,
+    ) -> dict:
+        host_and_insertion_ids = {"host_id": None, "host_entries": [], "insertion_ids": []}  # type: dict
+        ignored_specie_min_fraction = min([e.composition.get_atomic_fraction(ignored_specie) for e in entries])
+
+        for e in entries:
+            if e.composition.get_atomic_fraction(ignored_specie) == ignored_specie_min_fraction:
+                host_and_insertion_ids["host_entries"].append(e)
+            else:
+                host_and_insertion_ids["insertion_ids"].append(e.entry_id)
+        host_and_insertion_ids["host_id"] = min(host_and_insertion_ids["host_entries"],
```

Comment:
If I'm understanding this correctly, it looks like among the host entries, the one which is selected for the host_id is the oldest entry? Do we want less arbitrary criteria, like selecting the host entry with the lowest energy instead?

Reply:
Misread the code. Please disregard this comment!